pii_recognition/experiments/pii_validation/google_cloud_reports/mistakes_ner.json

```diff
@@ -0,0 +1,3580 @@
+{
+  "Please update billing addrress with Markt 84, M\u00dcLLNERN 9123 for this card: 5550253262199449": {
+    "billing": { "type": "OTHER", "score": 0.0, "src": "predicted" },
+    "card": { "type": "OTHER", "score": 0.0, "src": "predicted" },
+    "Markt 84": { "type": "OTHER", "score": 0.0, "src": "predicted" },
+    "M\u00dcLLNERN": { "type": "OTHER", "score": 0.0, "src": "predicted" },
+    "84": { "type": "NUMBER", "score": 0.0, "src": "predicted" },
+    "9123": { "type": "NUMBER", "score": 0.0, "src": "predicted" },
+    "5550253262199449": {
+      "type": "CREDIT_CARD",
+      "score": 0.0,
+      "src": "ground_truth"
+    },
+    "Markt 84, M\u00dcLLNERN 9123": {
+      "type": "LOCATION",
+      "score": 0.0,
+      "src": "ground_truth"
+    }
+  },
+  "My name appears incorrectly on credit card statement could you please correct it to Ms. Aybika Rushisvili?": {
```

Comment:
![image](https://user-images.githubusercontent.com/2088399/99197703-2ca77500-27e8-11eb-846c-6ccf100a2237.png)
This seems to do ok on https://cloud.google.com/natural-language#natural-language-api-demo

Reply:
This is the raw protobuf results I got (before passing to any parsers)

```
entities {
  name: "name"
  type_: OTHER
  salience: 0.6515209674835205
  mentions {
    text {
      content: "name"
      begin_offset: 3
    }
    type_: COMMON
  }
}
entities {
  name: "Aybika Rushisvili"
  type_: PERSON
  salience: 0.18994392454624176
  mentions {
    text {
      content: "Aybika Rushisvili"
      begin_offset: 88
    }
    type_: PROPER
  }
  mentions {
    text {
      content: "Ms."
      begin_offset: 84
    }
    type_: COMMON
  }
}
entities {
  name: "credit card statement"
  type_: OTHER
  salience: 0.15853510797023773
  mentions {
    text {
      content: "credit card statement"
      begin_offset: 31
    }
    type_: COMMON
  }
}
language: "en"
```

Correct predictions like `Aybika Rushisvili` is ignored and not logged into mistakes. However, the word `Ms.` based on the proto results is a PERSON. It seems to be a pattern that the demo is only showing the first `mentions`, including another example you commented below.

Reply:
ah ok. that makes sense. Sorry, I missunderstood that the correct predictions don't get logged. Makes sense now :-D